x-pack/plugins/ml/common/constants/settings.ts

```diff
@@ -5,3 +5,5 @@
  */
 
 export const FILE_DATA_VISUALIZER_MAX_FILE_SIZE = 'ml:fileDataVisualizerMaxFileSize';
+export const ANOMALY_DETECTION_ENABLE_TIME_RANGE = 'ml:anomalyDetectionDefaultTimeRangeEnable';
+export const ANOMALY_DETECTION_DEFAULT_TIME_RANGE = 'ml:anomalyDetectionDefaultTimeRangeSet';
```

Comment:
Is "set" necessary here? If not, I think it could be shortened like this:
```suggestion
export const ANOMALY_DETECTION_DEFAULT_TIME_RANGE = 'ml:anomalyDetectionDefaultTimeRange';
```

Reply:
I think I did it this way so the settings are in order. Although let me check with the Kibana team to see if there's a better way 🤔 .